CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(BioCell)
 
-set(BioCell_LIBRARIES BioCell)
```

Comment:
This calls essentially tells the ITK build system that this module has source (it is not header-only). And removal of this is causing build failure.

Reply:
True. Abandoning.